planner/languages/java/java_planner.go

```diff
@@ -56,19 +58,24 @@ func (p *Planner) GetPlan(srcDir string) *plansdk.Plan {
 		return plan.WithError(err)
 	}
 	installStage := p.installCommand(srcDir)
+	buildCommand := p.buildCommand()
 	return &plansdk.Plan{
 		DevPackages: []string{
 			defaultMaven,
 			javaPkg,
+			binUtils,
 		},
 		RuntimePackages: []string{
-			defaultMaven,
-			javaPkg,
+			binUtils,
```

Comment:
just curious, does `glibc` package work as a replacement?

For Rust's image, that had a smaller size than binutils

Reply:
@savil I tried but I got the same error as before adding `binutils`
> Error: java.io.IOException: Cannot run program "objcopy": error=2, No such file or directory

Reply:
could you try `jlink` with `--strip-java-debug-attributes` instead of `--strip-debug`?

Looks like that may fix this issue.
https://github.com/docker-library/openjdk/issues/351#issuecomment-524470228